eth-contracts/README.md

```diff
@@ -10,3 +10,10 @@ $ cd force-bridge-eth/eth-contracts
 $ npm install
 $ npx buidler test
 ```
+## Test on Testnet
+```bash
+# create .env file for testnet-ropsten, make sure you have enough ethers in your Ropsten account for delopy the contract
+$ echo ROPSTEN_DEPLOYER_PRIVATE_KEY=\"0x0000000000000000000000000000000000000000000000000000000000000000\"\\r\\n\
```

Comment:
Add a file `.env.example` may be better.
You can add the command to `package.json`.

Reply:
Just use `cp .env.example .env` here?